travis-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+dotnet test -c Debug -f netcoreapp1.0 MoreLinq.Test
```

Comment:
Indeed! I'll remove that part from the script and rename the script to `travis-mono-test.sh`. The only reason for its existence is that I did not manage to properly encode the commands in yaml syntax